packages/engine/Source/Scene/Scene.js

```diff
@@ -512,6 +514,8 @@ function Scene(options) {
    */
   this.cameraEventWaitTime = 500.0;
 
+  this.atmosphere = new Atmosphere();
```

Comment:
I believe this warrants:
* Inline documentation (Please include a code example if applicable)
* Updates to `CHANGES.md`

Reply:
I did mention `scene.atmosphere` in the changelog entry, is this big enough to warrant a separate changelog entry?